python/az/aro/azext_aro/custom.py

```diff
@@ -42,7 +43,11 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
                client_secret=None,
                pod_cidr=None,
                service_cidr=None,
+               software_defined_network=None,
+               disk_encryption_set=None,
```

Comment:
API Accepts separate disk encryption sets for master and workers, should we not add 2 options here too?

Reply:
@mjudeikis API does accept two valeus, but we [decided](https://github.com/Azure/ARO-RP/pull/1595#discussion_r671174070) to force worker encrpyion set id match master one for now (see code below).

https://github.com/Azure/ARO-RP/blob/1af1f2b0b4e6d9b6cdf346fd591157108c9478f8/pkg/api/v20210901preview/openshiftcluster_validatestatic.go#L292-L294

I don't worry much about implementation: I'm ok with `aro_create` accepting two separate params.

But if we are talking about exposing separate CLI params to the customers - I would be in favour of keeping 1 param for now to avoid confusion. If we decide to relax this validation - we will add a second param (`--disk-encryption-set` and new `--master-disk-encryption-set`) or rename existing and add new one (so we have, for example, `--worker-disk-encryption-set` and `--master-disk-encryption-set`).

Reply:
If we decide to relax I'd leave the old --disk-encryption-set, and add 2 new (one for master, one for worker). Then throw an error if the old one is specified at the same time as one of the new ones. This will keep backwards compatibility if someone uses az aro in a script.